src/pages/reservation-detail/ReservationDetail.tsx

```
@@ -0,0 +1,83 @@
+      <Navigation
+        text='상세예약'
+        leftIcon={<Icon name='ic_back' />}
+        handleLeftClick={handleClickBack}
+        rightIcon={
+          <>
+            <Button
+              onClick={handleDownload}
+              children={
+                <>
+                  <Icon
+                    name='ic_download'
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)